chapters/ch20.md

```diff
@@ -1,49 +1,57 @@
 # 20. Bias and Variance: The two big sources of error
 ->
-
+# 20. Độ chệch và phương sai: Hai nguồn lớn của lỗi
 
 Suppose your training, dev and test sets all come from the same distribution. Then you should always try to get more training data, since that can only improve performance, right?
 ->
-
+Giả sử khi huấn luyện, tập phát triển và tập kiểm tra có cùng phân phối. Khi đó bạn luôn nên cố gắng thu thập thêm dữ liệu, vì điều đó chỉ giúp cải thiện hiệu quả, đúng không? (cần dịch lại)
```

Comment:
"tập phát triển và tập kiểm tra *có* cùng phân phối" ----> "tập phát triển và tập kiểm tra *được lấy mẫu* cùng phân phối"

Reply:
Nếu dịch là "lấy mẫu cùng phân phối" thì không chuẩn. Vì "lấy mẫu" chỉ được dùng khi nói về một điểm dữ liệu, không phải một tập hợp.

Câu "come from the same distribution" tương đương với "have the same distribution", mình dịch là "có cùng phân phối" là rất sát.

Reply:
Em có đề xuất dịch lại thành:
"Khi đó bạn luôn ~~nên~~ *cần* cố gắng thu thập thêm dữ liệu, vì *dù sao* điều đó *cũng* chỉ giúp cải thiện hiệu quả, đúng không?"

Để trơn tru hơn.

Với lại sao "training data" anh lại chỉ dịch là "dữ liệu" thôi vậy?

Reply:
Anh dịch thiếu từ "training", anh sẽ bổ sung.